### ISSUE TITLE
Add <main> tags

### ISSUE THREAD

**Issue by ewmcelvaney** [CodefortheCarolinas/tax-benefit-website#53]
Users of the site may use tools that require special tags that indicate where the main content of the website is. To comply with WCAG, we need to add a main tag at the start of the content under the black heading/nav bar and a closing main tag right before  the closing body tag. The opening tag should say main role="main".
